GovUk.Frontend.AspNetCore.Extensions/Validation/RegisteredCharityNumberAttribute.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace GovUk.Frontend.AspNetCore.Extensions.Validation
+{
+    public class RegisteredCharityNumberAttribute : RegularExpressionAttribute
+    {
+        private const string SIX_NUMBERS_ONLY = @"(^[0-9]{6}$)";
+        private const string SEVEN_NUMBERS_ONLY = @"(^[0-9]{7}$)";
+        private const string SC_PREFIX_AND_SIX_NUMBERS = @"(^[Ss][Cc][0-9]{6}$)";
+
+        public RegisteredCharityNumberAttribute()
+            : base($@"{SIX_NUMBERS_ONLY}|{SEVEN_NUMBERS_ONLY}|{SC_PREFIX_AND_SIX_NUMBERS}")
+        {
+            ErrorMessage = "Enter a valid registered charity number";
```

Comment:
6. 'Enter a valid registered charity number' should be 'Enter a registered charity number in the correct format, like 123456, SC123456 or 1234567'. That affects the screenshot too.

Reply:
Updated the default error message